### PR TITLE
Format availability for the new style

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Adapts the template to display availability in a new way. No CSS used, simple boostrap alert is used.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | 

### Related core PRs
- https://github.com/PrestaShop/PrestaShop/pull/37018 Moves "Available with different options" to a new array key.
- https://github.com/PrestaShop/PrestaShop/pull/37013 Allows to distinguish between in stock and on backorder, so we can display it in green.

### How it looks
In stock
stock

Available on backorder
available_backorder

Not available (+ a new availability submessage)
not available

### CODE REVIEW

File: templates/catalog/_partials/product-add-to-cart.tpl
Comment (kpodemski) [inline templates/catalog/_partials/product-add-to-cart.tpl:9]: With all the logic out here this could be separated into another file, what do you think

Comment (Hlavtox) [reply templates/catalog/_partials/product-add-to-cart.tpl:9]: I would do it properly when somebody remakes all this `page-refresh-cant-move-anything-crap` :-)

**Review by ShaiMagal — APPROVED** [review]

**Comment by Hlavtox** [comment]
@florine2623 This can be tested standalone, the `availability_subtext` is optional.

**Review by florine2623 — COMMENTED** [review]
Hello @Hlavtox ,

I have a rebase going on when I install this PR. 
Am I missing something ?

Screenshot 2024-11-13 at 10 54 46

Waiting for your feedback ^^
Thanks!

**Comment by Hlavtox** [comment]
@florine2623 No idea, but the core doesn't have anything to do with this PR. :-) I suggest using a built version of hummingbird and just changing the one file manually.

**Review by florine2623 — APPROVED** [review]
Hello @Hlavtox ,

This has been tested with https://github.com/PrestaShop/PrestaShop/pull/37018.

As I couldn't install the PR properly, I followed your suggestion to change just the file changed on my template.

More details on the test of this PR : https://github.com/PrestaShop/PrestaShop/pull/37018#pullrequestreview-2472897398

Thanks !
It is QA ✅

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
